src/utils/envVars.ts

```diff
@@ -1,29 +1,23 @@
-// private vars
-export const PORTIS_DAPP_ID: string =
-  process.env.REACT_APP_PORTIS_DAPP_ID || '';
-export const INFURA_PROJECT_ID: string =
-  process.env.REACT_APP_INFURA_PROJECT_ID || '';
+export const IS_MAINNET                 = Boolean(process.env.REACT_APP_IS_MAINNET === 'true');
 
-// non-private
-export const WEB3_POLLING_INTERVAL = 12000;
-export const RPC_URL_GOERLI = `https://goerli.infura.io/v3/${INFURA_PROJECT_ID}`;
-export const RPC_URL_MAINNET = `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`;
-export const FORTMATIC_KEY = 'pk_test_D113D979E0D3508F';
-export const ALETHIO_URL = 'https://explorer.goerli.aleth.io/tx';
-export const ETHERSCAN_URL = 'https://goerli.etherscan.io/tx';
-export const CONTRACT_ADDRESS = '0x344b3a521ded954b4fa9ec8cc1d999631c998daa';
-export const MAINNET_ETH_REQUIREMENT = 524288;
-export const IS_MAINNET = false;
+// private vars (or derived from)
+export const PORTIS_DAPP_ID             = process.env.REACT_APP_PORTIS_DAPP_ID     || '';
+export const INFURA_PROJECT_ID          = process.env.REACT_APP_INFURA_PROJECT_ID  || '';
+export const ENABLE_RPC_FEATURES        = Boolean(INFURA_PROJECT_ID && INFURA_PROJECT_ID !== '');
+export const INFURA_URL                 = `https://${IS_MAINNET ? "mainnet" : "goerli"}.infura.io/v3/${INFURA_PROJECT_ID}`;
 
-// eth2.0 specs constants
-export const BLS_WITHDRAWAL_PREFIX = Buffer.from('00', 'hex');
-export const DOMAIN_DEPOSIT = Buffer.from('03000000', 'hex');
-export const EMPTY_ROOT = Buffer.from(
-  '0000000000000000000000000000000000000000000000000000000000000000',
-  'hex'
-);
-export const GENESIS_FORK_VERSION = Buffer.from('00000000', 'hex');
-export const GWEI = 1e9;
-export const MIN_DEPOSIT_AMOUNT = 1 * GWEI;
-export const MAX_DEPOSIT_AMOUNT = 32 * GWEI;
-export const PRICE_PER_VALIDATOR = MAX_DEPOSIT_AMOUNT / GWEI;
+// public
+export const ALETHIO_URL                = IS_MAINNET ? 'https://explorer.goerli.aleth.io/tx' : 'https://explorer.goerli.aleth.io/tx';
+export const ETHERSCAN_URL              = IS_MAINNET ? 'https://goerli.etherscan.io/tx' : 'https://goerli.etherscan.io/tx';
+export const FORTMATIC_KEY              = process.env.REACT_APP_FORTMATIC_KEY       || 'pk_test_D113D979E0D3508F';
+export const CONTRACT_ADDRESS           = process.env.REACT_APP_CONTRACT_ADDRESS    || '0x344b3a521ded954b4fa9ec8cc1d999631c998daa';
+export const ETH2_NETWORK_NAME          = process.env.REACT_APP_ETH2_NETWORK_NAME   || '';
+export const ETH_REQUIREMENT            = process.env.REACT_APP_ETH_REQUIREMENT     || 524288;
+export const PRICE_PER_VALIDATOR        = process.env.REACT_APP_PRICE_PER_VALIDATOR || 32;
+
+// BLS signature verification variables
+export const GWEI                       = 1e9;
+export const MIN_DEPOSIT_AMOUNT         = 1 * GWEI;
```

Comment:
Nitpicking suggestion for readability:

```suggestion
export const ETHER_TO_GWEI              = 1e9;
export const MIN_DEPOSIT_AMOUNT         = 1 * ETHER_TO_GWEI;
```